macros/get_lookback_date_limits.sql

```diff
@@ -25,44 +19,37 @@
     {% if page_view_max == True %}
       {%- do exceptions.raise_compiler_error("Snowplow Warning: the derived.page_view source does not cover the full fractribution analysis period. Please process your web model first before proceeding with this package.") %}
     {% endif %}
+    {% set page_view_min = result[0][1] %}
+    {% if page_view_min == True %}
+      {%- do exceptions.raise_compiler_error("Snowplow Warning: the derived.page_view source does not cover the full fractribution analysis period. Please backfill / reprocess your web model first before proceeding with this package.") %}
+    {% endif %}
```

Comment:
Could we be a little more explicit here in terms of if their min or max date isn't covered?

Reply:
If we wanted we could also pull the min/max date and actually compare it to the variable, telling them the values so they know what they need to process to/from... up to you though that's more effort for little gain.